--- bootsplash-3.4.0/scripts/make-boot-splash-raw.xz_crc32~	2012-06-08 15:52:30.658016423 +0200
+++ bootsplash-3.4.0/scripts/make-boot-splash-raw	2012-06-08 15:52:32.187997297 +0200
@@ -20,7 +20,7 @@ for COMPRESSION in xz bz z uncompressed;
 	case $COMPRESSION in
 	xz)
 		DECOMPRESS=/usr/bin/xzcat
-		COMPRESS="/usr/bin/xz -9e"
+		COMPRESS="/usr/bin/xz --check=crc32"
 		;;
 	bz)
 		DECOMPRESS=/usr/bin/bzcat
